discordSplash/channel.py

```diff
@@ -1,10 +1,10 @@
 from dataclasses import dataclass
```

Comment:
For imports, decide whether there should be a whitespace between them, or no whitespace.

Reply:
fixed